src/Mauro/Greeter.test.ts

```diff
@@ -11,4 +11,21 @@ describe("Greeter.greet", () => {
     // assert
     expect(result).toBe("Hello Mauro");
   });
+
+  it("should trim received name", () => {
+    // arrange
+    const greeter = new Greeter();
+
+    // act
+    const result = greeter.greet("  Mauro  ");
+
+    // assert
+    expect(result).toBe("Hello Mauro");
+
+    // act
+    const result2 = greeter.greet("Mauro  ");
```

Comment:
An alternative could be to create two tests, one for each assert:
* it should trim the received name when the name has leading and trailing spaces
* it should trim the received name when the name has trailing spaces